content/onboarding/learners/welcome/_index.md

```diff
@@ -3,4 +3,24 @@ _db_id: 879
 content_type: topic
 ready: true
 title: Welcome to your learnership
----
\ No newline at end of file
+---
+
+![congratulations pic](Congratulations_pic.png)
+
+##**Who are we and what do we do?**
+
+At the heart of Umuzi is our energy, our commitment and passion to reduce social inequality in Africa through digital education. We are excited about what we do, who we work with and the value we are adding on a daily basis. We do this with warmth, empathy, and dynamism. We are a bold, daring and exciting organisation, creating meaning through sharing our energy - it’s contagious!
+
+Through our energy we create connection with individuals that enables a shift to occur, causing a ripple effect, creating communities. This ripple effect has an impact on the people we meet, we work with, partner with, and then their families and friends, not just in the present, but in the future too. 
+
+The young people (YOU) that move through our programmes are at the heart of what we do at Umuzi, and it is their journey and stories that drive us to continue on this mission.
+
+Join us in being part of this movement!
+
+**ENERGY, CONNECTION, IMPACT**
+
+#**Now for a little bit of admin before we move forward!**
+
+**If you haven’t** completed the payroll information form, the baeline survey and/or submitted your certified documents just yet, [click here]( https://docs.google.com/document/d/18dKQlxM8CMWaP0MJAf5PXdi_mqg_cNfUePiOD_YjhSo/edit) to do so before continuing to the next card. 
```

Comment:
remember this is public, do we want anyone anywhere to be able to access this or should this page reference something that is only accessable internally, i.e should we move the links to forms to the umuzi site?

Reply:
I thgink we should add the forms + umuyzi learner agreement on this sites page

Reply:
Hi Ry, I had linked the payroll info and docs links to a separate sheet with access restricted to people who have an Umuzi email address. https://docs.google.com/document/d/18dKQlxM8CMWaP0MJAf5PXdi_mqg_cNfUePiOD_YjhSo/edit

Is this a viable solution?

If not, that's okay. I've removed that restricted link and replaced it with a task list and message telling them to go back to my initial email where they will find links to the forms, etc.